homeassistant/components/assist_satellite/entity.py

```diff
@@ -262,6 +262,8 @@ async def async_internal_start_conversation(
             raise SatelliteBusyError
 
         self._is_announcing = True
+        self._set_state(AssistSatelliteState.RESPONDING)
```

Comment:
Please make a separate PR for the base entity changes.

Reply:
Is this a fix?

Reply:
Split out here: https://github.com/home-assistant/core/pull/141388